packages/patterns/src/types.js

```diff
@@ -456,36 +456,75 @@ export {};
  * Matches any Passable that is matched by `subPatt` or is the exact value `undefined`.
  */
 
+/**
+ * @typedef {'never' | 'passable' | 'raw'} DefaultGuardType
+ */
+
+/**
+ * @typedef {<M extends Record<PropertyKey, MethodGuard>>(
+ *   interfaceName: string,
+ *   methodGuards: M,
+ *   options: {defaultGuards?: 'never', sloppy?: false }) => InterfaceGuard<M>
```

Comment:
Should this be

```suggestion
 *   options?: {defaultGuards?: 'never', sloppy?: false }) => InterfaceGuard<M>
```

Reply:
Actually, no.  These two overloads are only used if the options bag is specified.